spec/controllers/tasks_controller_spec.rb

```diff
@@ -921,6 +921,15 @@
       get_request
       expect(response).to render_template('export_external_start')
     end
+
+    context 'when account_link is not owned by user' do
```

Comment:
I would prefer to go with a human description here and in the following:

```suggestion
    context 'when the account link is neither owned by nor shared with the requesting user' do
```